src/sentry/rules/filters/assigned_to.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+
+from django import forms
+
+from sentry.rules.filters.base import EventFilter
+from sentry.mail.actions import MemberTeamForm
+
+
+class AssigneeTargetType:
+    NO_ONE = "NoOne"
```

Comment:
nit: Maybe use UNASSIGNED and map to "No One" for display to keep assigned enum consistent with the status of the group